### PR TITLE
Console Clear

### PR DESCRIPTION
**Pull request by AlCzSa**

### CODE REVIEW

**Comment by bcremer** [comment]
Hey @AlCzSa, thanks for the PR, but what is the pupose of this commands?

**Comment by AlCzSa** [comment]
Hi @bcremer, i use the whole day a terminal/ console in Linux. And if I use your shopware shell, I miss sometimes these commands. That's why i've added theses

**Comment by bcremer** [comment]
So are you are using the shopware console in shell mode like `./bin/shopware --shell`? 
That`s a possible usecase for that commands, indeed.

**Comment by AlCzSa** [comment]
Hi @bcremer, yeah, i use the consle mostly in shell mode.

File: engine/Shopware/Commands/ExitCommand.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Shopware\Commands;
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
```
Comment (bcremer) [inline engine/Shopware/Commands/ExitCommand.php:10]: Please update the comment.

Comment (AlCzSa) [inline engine/Shopware/Commands/ExitCommand.php:10]: Updated, thanks.

**Comment by dnoegel** [comment]
Hm, note sure about this one - aren't CTRL+D and CTRL+L quite fine? Or is there another use case?

**Comment by AlCzSa** [comment]
For the most people, sure. But exit and clear gets me every time. That's why i've added these commands for me. After that, I decided to make a pull request for that. Maybe I am not the only one..

**Comment by bcremer** [comment]
Thank you @AlCzSa for the PR, we appreciate your work.

But after considering the comment from @dnoegel and due to the fact that also the symfony standard edition does not provide such commands we decided to not include this PR for now.

**Comment by AlCzSa** [comment]
@bcremer , @dnoegel no problem, you're welcome

**Pull request closed without merging**